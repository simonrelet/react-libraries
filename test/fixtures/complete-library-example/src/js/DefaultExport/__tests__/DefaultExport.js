import { render } from '@testing-library/react'
import React from 'react'
import DefaultExport from '../DefaultExport'

describe('<DefaultExport />', () => {
  it('should render the component', () => {
    const { container } = render(<DefaultExport />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
