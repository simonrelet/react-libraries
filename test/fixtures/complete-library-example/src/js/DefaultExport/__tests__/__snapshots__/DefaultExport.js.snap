// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DefaultExport /> should render the component 1`] = `
<div>
  DefaultExport
</div>
`;
